tensornetwork/tn_keras/test_layer.py

```diff
@@ -1,11 +1,14 @@
 import pytest
 import numpy as np
+import math
 from tensorflow.keras import backend as K
 from tensorflow.keras.models import Sequential
 from tensornetwork.tn_keras.dense import DenseDecomp
+from tensornetwork.tn_keras.mpo import DenseMPO
+from tensorflow.keras.layers import Dense  # type: ignore
 
 
-@pytest.fixture(params=[50, 100, 256])
+@pytest.fixture(params=[256, 1024])
```

Comment:
Some other tests to add:

-> cls.from_config(layer.get_config()) should return an identically configured layer. (The weights will be different, but you should be able to call set_weights(get_weights()) to get an identical layer.
-> model.save() should not die. (This is required for Tensorflow Serving, which I think will be a huge win with this layer.)

Reply:
Agreed! Added `test_config` and `test_model_save` to address these.